kinetic/distribution.yaml

```diff
@@ -2464,6 +2464,23 @@ repositories:
       url: https://github.com/tork-a/dynpick_driver.git
       version: master
     status: maintained
+  e100_base:
+    doc:
+      type: git
+      url: https://github.com/gaitech-robotics/e100_base.git
```

Comment:
This repository does not appear to exist. Maybe it's private?

Reply:
It's private now. Is it necessary to change to public to register?

Reply:
It's necessary to be public to be indexed.

If it's private we can't check out the source code and generate the documentation, nor run continuous integration. Note that your release repository contains snapshots of the repository at the tag released so it can be built on the buildfarm so keeping the upstream repository private doesn't make a lot of sense.

Reply:
I'll request it again after modification.